### DISCUSSION TITLE
How to create filter condition when some cell is selected ?

### DISCUSSION THREAD

**Discussion by sergeynilov** [dbeaver/dbeaver#16062] [Q&A]
Hello,
Can I in dbeaver 21.2.1 when some cell is selected in grid of data by some action

to create filter condition like `day` = '2022-03-12' is cell with such column/value was selected?
Thanks !

**Comment by uslss** [comment]
If I understand you right, you could use the filter button in the Column header while the needed cell is selected in Data Viewer. More about filters could be learned [here](https://github.com/dbeaver/dbeaver/wiki/Data-Filters)
![dbeaver_K3Sjuz5j5s](https://user-images.githubusercontent.com/31996417/158569094-4718995a-ed32-47ce-9f2f-5aa1ad52f8b3.gif)
